README.md

```diff
@@ -296,24 +296,19 @@ This will build the controller image and then push it as the image defined in `I
 
 _WARNING: This step requires the working OCI Registry, Cluster and Kyma CLI from our [Pre-requisites](#pre-requisites)_
 
-1. The module operator manifests from the `default` kustomization (not the controller image) will now be bundled and pushed.
-   Assuming the settings from [Prepare and build module operator image](#prepare-and-build-module-operator-image) for single-cluster mode, and assuming the following module settings:
+1. **Prerequisites**
+   
+   Generate the CRDs for the module from the `default` kustomization into a manifest file using the following command:
```

Comment:
The `build-manifests` is not only generate CRDs, but all the resources related to this module, you can also mention the output for this command, it's `template-operator.yaml`, which offers to `manifest` in module-config.yaml